src/en/users-sample-commands.md

```diff
@@ -0,0 +1,81 @@
+Title: Juju sample commands
```

Comment:
Is this page linked from anywhere? It may be a good idea e.g. to link it from the main users page.